filesystem/dirs_test.go

```diff
@@ -14,21 +14,19 @@ import (
 var RootFolder = "/"
 
 func TestPathExists(t *testing.T) {
-	tempFile := filepath.Join(os.TempDir(), "testdir"+uuid.New().String()+"_"+t.Name())
+	tempFile := filepath.Join(t.TempDir(), uuid.New().String()+"_"+t.Name())
 	_, err := os.Create(tempFile)
 	require.NoError(t, err, "couldn't create temp path")
 	assert.True(t, PathExists(tempFile), "expecting existence of path")
-	os.RemoveAll(tempFile)
 }
 
 func TestGetFullDirectoryPath(t *testing.T) {
-	tempFile := filepath.Join(os.TempDir(), "testdir"+uuid.New().String()+"_"+t.Name())
+	tempFile := filepath.Join(t.TempDir(), "testdir"+uuid.New().String()+"_"+t.Name())
 	err := os.MkdirAll(tempFile, os.ModeDir)
```

Comment:
Considering that the tested function _creates_ the directory if it doesn't exist:
>// GetFullDirectoryPath gets the OS specific full path for a named directory.
>// The directory is created if it doesn't exist.

It would be better to skip creating it in the test and verify if the function created it as it promises.

Btw, this function's name doesn't match its responsibilities.